### PR TITLE
Maistra Compatibility

### PR DESCRIPTION
**Pull request by jshaughn**

### CODE REVIEW

File: content/documentation/staging/installation-guide/index.adoc [hunk 1]
```diff
@@ -49,6 +49,22 @@ Kiali requires a supported version of Istio. link:https://istio.io/news/[The Ist
...
 
+[cols="15%,15%,70%",options="header"]
+|===
+|Maistra
+|Kiali
+|Notes
+
+| 2.0
+| 1.24
+| Maistra 2.0 must be installed using Kiali Operator for OpenShift, v1.36. Other operator versions are not compatible.
```
Comment (jmazzitelli) [inline content/documentation/staging/installation-guide/index.adoc:60]: > Kiali Operator for OpenShift
I don't know what that means "for OpenShift"

Comment (jshaughn) [reply content/documentation/staging/installation-guide/index.adoc:60]: that was supposed to mean, not "community"

Comment (jmazzitelli) [reply content/documentation/staging/installation-guide/index.adoc:60]: Gonna need something like:

[cols="15%,15%,15%,70%",options="header"]
|===
|Maistra
|SMCP CR
|Kiali
|Notes

|1.1
|1.1
|1.12
|To install a service mesh control plane version 1.1 using Maistra 1.1 requires Kiali Operator v1.36 installed. Other operator versions are not compatible.


|2.0
|1.1
|1.12
|To install a service mesh control plane version 1.1 using Maistra 2.0 requires Kiali Operator v1.36 installed. Other operator versions are not compatible.

|2.0
|2.0
|1.24
|To install a service mesh control plane version 2.0 using Maistra 2.0 requires Kiali Operator v1.36 installed. Other operator versions are not compatible.

Comment (jmazzitelli) [reply content/documentation/staging/installation-guide/index.adoc:60]: To be complete (but no one is using SMCP 1.0 anymore, so.. really, this is superflous):

|1.1
|1.0
|n/a
|Installing a service mesh control plane version 1.0 using Maistra 1.1 is not supported by Kiali Operator.

**Review by jmazzitelli — APPROVED** [review]
This looks correct to me.

**Pull request merged**